question17/main.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const imgListsWrapper = document.getElementById('js-imgListsWrapper');
+const imgLists = document.getElementById('js-imgLists');
+
+const showLoadImg = () => {
+  const loadImg = createElementWithClassName("img", "loading");
+  loadImg.src = "./img/loading-circle.gif";
+  loadImg.id = 'loading';
+  imgListsWrapper.appendChild(loadImg);
+} 
+
+const removeLoadImg = () => {
+  document.getElementById('loading').remove();
+} 
+
+const getData = new Promise((resolve) => {
+  showLoadImg();
+  setTimeout(() => {
+    resolve(fetch('https://mocki.io/v1/09593e78-80ba-4765-940e-3c70fb46078c'));
+  }, 3000);
+})
+
+//Get json data
+async function callApi() {
```

Comment:
```suggestion
const callApi = async () => {
```

Function literals are often used, why are function declarations used here?
I might write it as above.

I'm sorry if I'm wrong

Reply:
Thank you for the review! I had not noticed that at all, but you are right. I thought it would be better to align the function definitions, so I fixed it. Please confirm.
fixed commit: https://github.com/csakyo/homework/pull/31/commits/426f23250e813bfb10ae71c67a386b3ecaa34f10